src/axolotl/utils/schemas/validation.py

```diff
@@ -97,6 +97,16 @@ def check_test_datasets_bench(cls, data):
             data["test_datasets"] = [{"path": "axolotl-ai-co/empty-test-ds"}]
         return data
 
+    @model_validator(mode="before")
+    @classmethod
+    def default_pad_to_sequence_len_value(cls, data):
+        if (
+            data.get("sample_packing") is not None
+            and data.get("pad_to_sequence_len") is None
+        ):
+            data["pad_to_sequence_len"] = data["sample_packing"]
```

Comment:
fyi, we have another validator doing almost the same thing https://github.com/axolotl-ai-cloud/axolotl/blob/170322a1f0e099bd8e184d2d352cf2f60437527c/src/axolotl/utils/schemas/validation.py#L206-L220